src/components/KeyAgencyActions.vue

```diff
@@ -68,7 +68,7 @@ export default Vue.extend({
       await this.loadAnswer();
     },
     submitAnswer: async function () {
-      await util.saveYesNoAnswer(this.answer[0].id, this.displayedJustification, this.yesNoValue === this.items[0] ? "Y" : "N", this.$store.state.token);
+      await util.saveYesNoAnswer(this.answer[0].id, this.displayedJustification, this.yesNoValue, this.$store.state.token);
```

Comment:
We should be consistent code and user's input.
```suggestion
      await util.saveAgreementAnswer(this.answer[0].id, this.displayedJustification, this.agreementValue, this.$store.state.token);
```